vue-frontend/src/components/contact/ContactForm.vue

```diff
@@ -1,301 +1,135 @@
+              class="custom-text-input v2-title-2-text"
               type="text"
               name="email"
               required
+              autocomplete="given-email"
               v-model="email"
               :disabled="disableInput"
+              placeholder="Your email"
             />
-            <span v-if="showValidationError" class="error email-error"
+            <span v-if="showValidationError" class="error tw-text-red-600"
               >This field is required</span
             >
           </div>
-          <br />
-          <div class="my-3">
+          <div class="tw-pt-5 lg:tw-pt-14">
             <textarea
-              class="input-textarea"
-              name="message"
-              rows="10"
-              :v-bind="message"
-              v-model="message"
-              placeholder="I have a message or information for Canopas team."
+              class="custom-text-input v2-title-2-text"
+              name="project"
+              rows="1"
+              autocomplete="given-project-info"
+              v-model="projectInfo"
               :disabled="disableInput"
+              placeholder="Tell us about your project"
             ></textarea>
           </div>
-          <div>
-            <div v-if="contactType < 2" class="radio-buttons mb-3">
-              <label class="contactLabel1">
-                <input
+          <div class="tw-pt-5 lg:tw-pt-10">
+            <input
+              class="custom-text-input v2-title-2-text"
+              type="text"
+              name="reference"
+              required
+              autocomplete="given-reference"
+              v-model="reference"
+              :disabled="disableInput"
+              placeholder="How did you find us?"
+            />
+            <span v-if="showValidationError" class="error tw-text-red-600"
+              >This field is required</span
+            >
+          </div>
+
+          <div class="tw-py-16">
+            <div class="tw-text-slate-400 v2-title-2-text">
+              What's your preferred mode of communication?
+            </div>
+            <div class="tw-flex tw-flex-col tw-w-fit sm:tw-flex-row tw-pt-6">
+              <label class="contactLabel1"
+                ><input
+                  class="tw-hidden"
                   type="radio"
                   name="contact_type"
-                  value="0"
+                  value="2"
                   v-model.number="contactType"
                   :disabled="disableInput"
                 />
-                <div class="box">
-                  <span class="contact-form-text"
-                    >If you love to talk, You can schedule a meeting with our
-                    team member according to your convenience from here.</span
-                  >
-                </div>
-              </label>
-              <label class="contactLabel2">
-                <input
+                <div class="contact-button tw-mb-6 sm:tw-mr-9 sm:tw-mb-0">
```

Comment:
What will happen if i click on this?

Reply:
And when we choose this button that time send the mail with `schedule meeting popup`

Reply:
Its not showing me any popup, when click